rpcserver.go

```diff
@@ -2581,6 +2581,27 @@ func (r *rpcServer) InsertProof(ctx context.Context,
 	return r.marshalIssuanceProof(ctx, req.Key, newUniverseState)
 }
 
+// Info returns a set of information about the current state of the Universe.
+func (r *rpcServer) Info(ctx context.Context,
+	req *unirpc.InfoRequest) (*unirpc.InfoResponse, error) {
+
+	universeStats, err := r.cfg.UniverseStats.AggregateSyncStats(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	uniServers, err := r.cfg.FederationDB.UniverseServers(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	return &unirpc.InfoResponse{
+		RuntimeId:      r.runtimeID,
+		NumAssets:      universeStats.NumTotalAssets,
+		NumSyncServers: uint32(len(uniServers)),
```

Comment:
I think let's remove this bit? Given this is information that would be shown which isn't leakd anywhere else in the set of calls.

Reply:
Okay, makes sense, removed.